### PR TITLE
Data/additions nov 2020

### PR DESCRIPTION
**Pull request by danieltwigt**
Changes related to addition of CRUCIAL and MSFD datasets.

See https://issuetracker.deltares.nl/projects/GDV/issues/GDV-433.

Back-end changes:

- Added new datasets to datasets and datasets_access files
- Added some of the scripts to prepare and upload datasets
Fixed flask-apispec to version 0.8.8. This was required to install all relevant packages on my new laptop (ran into conflicts with newer version of flask-apispec, and version 0.8.8 was used on my other laptop, so reverted to that one)

### CODE REVIEW

**Comment by evetion** [comment]
You've got a CI error based on the Python packages:
`error: apispec 2.0.2 is installed but apispec>=4.0.0 is required by {'flask-apispec'}`

File: datasets/crucial/evaporation_deficit_upload.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+from os.path import basename
+from os import environ
+import subprocess
+from datetime import datetime
+import glob
+
+from google.cloud import storage
+
+def upload_blob(bucket_name, source_file_name, destination_blob_name):
```
Comment (evetion) [inline datasets/crucial/evaporation_deficit_upload.py:11]: These upload functions are already in some other datasets, couldn't we re-use them?

Comment (evetion) [reply datasets/crucial/evaporation_deficit_upload.py:11]: This whole file could be replaced by a small README.md that explains:
- Source of data (P disk/project something/where to the .nc files come from)
- GEE upload path

File: datasets/crucial/groundwater_table_declining_trend.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                 'Groundwater_Table_Declining_Trend_2035-01-31_2054-12-31',
+                 'Groundwater_Table_Declining_Trend_2055-01-31_2065-12-31',
+                 'Groundwater_Table_Declining_Trend_2035-01-31_2044-12-31',
+                 'Groundwater_Table_Declining_Trend_2045-01-31_2054-12-31',
+                 'Groundwater_Table_Declining_Trend_2005-01-31_2065-12-31']
+
+    # write to one geotiff file
+    dst_filename = "Groundwater_Table_Declining_Trend.tif"
+
+    ii=0
```
Comment (evetion) [inline datasets/crucial/groundwater_table_declining_trend.py:36]: Use `enumerate`

**Review by evetion — APPROVED** [review]
In the datasets files, could you remove all commented out lines and print statements? It can be useful to put a `README.md` or detail in code where the netcdfs come from and whether they're a single batch, or constantly generated/uploaded.

**Comment by evetion** [comment]
Depends on https://github.com/openearth/hydro-engine-service/pull/46

File: datasets/crucial/evaporation_deficit_upload.py (hunk 1 above)
Comment (danieltwigt) [reply datasets/crucial/evaporation_deficit_upload.py:11]: Removes unneeded print statements and commented out lines. Regarding re-using upload functions already in use for other datasets: could (re)use utils.py as included in glossis folder for this, but might be useful to reorganize some of the files in \datasets in that case (move utils.py up one level?). Would suggest not to do this now, but to discuss how to best organize this first, and do in the next sprint where this functionality is relevant.

File: datasets/crucial/groundwater_table_declining_trend.py (hunk 2 above)
Comment (danieltwigt) [reply datasets/crucial/groundwater_table_declining_trend.py]: Changes to enumerate in the code

**Pull request merged**